willow/plugins/pillow.py

```diff
@@ -150,6 +153,28 @@ def set_background_color_rgb(self, color):
 
         return PillowImage(new_image.convert('RGB'))
 
+    @Image.operation
+    def optimize(self, image_file, format, quality=85, progressive=False):
+        for optimizer in self.get_optimizers_for_format(format):
+            optimizer.process(image_file, quality=quality, progressive=progressive)
+
+    def save_and_optimize(self, f, image, format, **kwargs):
```

Comment:
Suggestion: Rename this method to `save()` and make `optimize` an kwarg with a default value of `True`

Reply:
Rationale: We are always saving, but not always optimising. Also, it might be handy to retain a way to save without optimising even when optimisers are registered (for example, to compare results with/without optimisation)

Reply:
Since this is an instance method and `image` is almost always `self.image`, would it be worth ditching `image` from args here and looking for an override in kwargs instead? Line below could then be:

`image = kwargs.get("image") or self.image`